sdk/iot/hub/samples/README.md

```diff
@@ -23,7 +23,32 @@ This document describes how to use samples and what is done in each sample.
 
 ## Getting started
 
-Getting started explained in detail [here][SDK_README_GETTING_STARTED].
+These samples use X509 authentication to connect to IoT Hub. To easily run these samples, we have provided
+two scripts: one to generate the root cert and device cert used to connect, and the other to generate the
```

Comment:
Please remove the Intermediate Certificate Authority scenario - I think that is well outside of a sample. You could though link to IoT Hub / Provisioning documentation on what is required to achieve that scenario.